zone/client.h

```diff
@@ -1516,8 +1517,9 @@ class Client : public Mob
 
 	void SendReloadCommandMessages();
 
-	void SendItemRecastTimer(int32 recast_type, uint32 recast_delay = 0);
+	void SendItemRecastTimer(int32 recast_type, uint32 recast_delay = 0, bool ignore = false);
```

Comment:
`ignore` ignore what?

Reply:
ignore_casting_requirement but you asked me to change it :D

Reply:
At least prefix with `in_` it may sound like nit picky feedback, but it has caused uninitialized values in the past having same name variables

`in_ignore_casting_requirement`

Reply:
Sounds good, was just wondering what you meant